### PR TITLE
build(nebula_examples): add ament_lint dependencies

### PR DESCRIPTION
**Pull request by esteve**
## PR Type

- Bug Fix

## Description

`ament_lint` is called from `CMakeLists.txt` but it's not declared as a dependency in `package.xml`, this PR addresses that.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

**Comment by esteve** [comment]
@amc-nu ping :slightly_smiling_face:

File: nebula_examples/package.xml [hunk 1]
```diff
@@ -28,6 +28,9 @@
   <depend>velodyne_msgs</depend>
   <depend>yaml-cpp</depend>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```
Comment (esteve) [inline nebula_examples/package.xml:32]: Packages in Autoware use `autoware_lint_common` here, but I wasn't sure if Nebula can depend on Autoware, @amc-nu thoughts?

Comment (amc-nu) [reply nebula_examples/package.xml:32]: Sorry I didn’t catch you there
> but I wasn't sure if Nebula can depend on Autoware

Add Autoware as a dependency?

Comment (amc-nu) [reply nebula_examples/package.xml:32]: I see. Thank you for clarifying.
Yes, the plan is to keep it independent.

**Comment by esteve** [comment]
Closing. Superseded by #67

**Pull request closed without merging**